girderformindlogger/api/v1/user.py

```diff
@@ -812,19 +812,21 @@ def getUsersDetails(self):
         .errorResponse(('You do not have write access for this user.',
                         'Must be an admin to create an admin.'), 403)
     )
+
```

Comment:
This (already deprecated) API call doesn't include the newly added `login` in the parameters for the interactive documentation.